specs/phase0/beacon-chain.md

```diff
@@ -732,16 +732,12 @@ def compute_shuffled_index(index: uint64, index_count: uint64, seed: Bytes32) ->
 
     # Swap or not (https://link.springer.com/content/pdf/10.1007%2F978-3-642-32009-5_1.pdf)
     # See the 'generalized domain' algorithm on page 3
-    for current_round in map(uint64, range(SHUFFLE_ROUND_COUNT)):
+    for current_round in range(SHUFFLE_ROUND_COUNT):
         pivot = bytes_to_int(hash(seed + int_to_bytes(current_round, length=1))[0:8]) % index_count
```

Comment:
Without the `map`, the `current_round` is `int`, it's still a bit buggy in `int_to_bytes(current_round, length=1)` typing.

https://github.com/ethereum/eth2.0-specs/pull/1935 is a more comprehensive solution.
Alternatively, a simpler solution is casting it:
```suggestion
        current_round = uint64(current_round)
        pivot = bytes_to_int(hash(seed + int_to_bytes(current_round, length=1))[0:8]) % index_count
```

Reply:
IMHO the `current_round` should be a single byte, and the max round count should reflect that. Since this is what is hashed in, and what is used by clients. We can figure out how to improve `compute_shuffled_index` further in #1935, and keep changes minimal here for now.